packages/pnpm/README.md

```diff
@@ -2,19 +2,19 @@
 
 ![](https://i.imgur.com/qlW1eEG.png)
 
-Fast, disk space efficient package manager:
+Fast, disk space-efficient package manager:
```

Comment:
It's a complicated case as the first part consists of two words. Any chance you could provide a reference to this grammar rule?